recipes/handout/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "handout" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "dbe5da9b422fa937b94a1a5221ce99387ebd75fe97ab4255e49b26d846b8614b" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py<35]
```

I think this recipe can be noarch and you can select the python which you need on requirements
```suggestion
  noarch: python
```